src/Application/Common/Results/ErrorCode.cs

```diff
@@ -32,6 +32,8 @@ public enum ErrorCode
     ItemBadSlot,
     ItemBroken,
     ItemDisabled,
+    ItemUpgradedVersionNotFound,
+    ItemNotUpgradable,
```

Comment:
Could you order the enum alphabetically, same for the associated method

Reply:
done